src/cpu.cpp

```diff
@@ -403,6 +403,7 @@ static bool find_temp_input(const std::string path, std::string& input, const st
     return false;
 }
 
+#ifndef MANGOAPP
```

Comment:
@flightlessmango keeps flip-flopping :D unnecessary now (but the value of find_fallback_temp_input itself is suspect).